src/views/Homepage/CharacterIntro/styles.module.css

```diff
@@ -37,7 +37,7 @@
   display: grid;
   grid-template-rows: repeat(auto, 1fr);
   grid-template-columns: repeat(2, 1fr);
-  grid-column-gap: var(--spacing-xxx-loose) var(--spacing-base);
+  gap: var(--spacing-base);
```

Comment:
short `gap` is the new standard replacing `grid-gap`

https://css-tricks.com/snippets/css/complete-guide-grid/#gapgrid-gap